.dockerignore

```diff
@@ -23,6 +23,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+.direnv/
```

Comment:
This directory is put by a program I use called `direnv`. It usually contains a Python environment that is too big to send to Docker daemon on every build.